nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -119,6 +122,19 @@ const DEFAULT_GATEWAY_CONFIG = {
   },
   tier: 'default',
 };
+const DEFAULT_DNS_CONFIG = {
+  enable_caching: false,
+  local_ttl: 0,
+  records: [],
+};
+const DEFAULT_HE_CONFIG = {
+  enable_encryption: false,
+  encryption_key: '',
+  enable_header_enrichment: false,
+  he_encoding_type: 'BASE64',
+  he_encryption_algorithm: 'RC4',
+  he_hash_function: 'MD5',
+};
```

Comment:
I get that these defaults are only used in this file, but it's been uncomfortably large for a while already. Could you split these constants off to a separate file?

Reply:
Including the `DEFAULT_GATEWAY_CONFIG` immediately above the lines specified in this comment.